nemo_skills/evaluation/metrics/rasb_metrics.py

```
@@ -0,0 +1,357 @@
+        return sorted_values[lower] * (1 - weight) + sorted_values[upper] * weight
+
+    def _compute_aggregate_stats(self) -> dict[str, float]:
+        """
+        Compute aggregate statistics across all environments.
+
+        Returns:
+            Dictionary with mean, median, Q1, Q3, std, and overall pass rates.
+        """
+        # Collect per-environment pass rates
+        pass_rates = []
+        for env_id, counter in self.by_env_id.items():
```

Ruff (0.15.11)

[warning] Loop control variable `env_id` not used within loop body

Rename unused `env_id` to `_env_id`

(B007)